framework/elements/Button.js

```diff
@@ -0,0 +1,24 @@
+const BaseElement = require('./BaseElement');
+
+class Button extends BaseElement {
+    constructor(name, selector) {
+        super(name, selector);
+    }
+
+    click() {
```

Comment:
please move all these methods to the parent class since it is possible to click not only button bu other elements as well